src/main/java/com/vinylteam/vinyl/service/impl/DefaultUserService.java

```diff
@@ -1,6 +1,41 @@
 package com.vinylteam.vinyl.service.impl;
 
+import com.vinylteam.vinyl.dao.jdbc.JdbcUserDao;
+import com.vinylteam.vinyl.entity.User;
 import com.vinylteam.vinyl.service.UserService;
 
+import java.util.Optional;
+
 public class DefaultUserService implements UserService {
+
+    private final JdbcUserDao jdbcUserDao;
+
+    public DefaultUserService() {
+        this.jdbcUserDao = new JdbcUserDao();
+    }
+
+    @Override
+    public boolean add(User user) {
+        return jdbcUserDao.add(user);
+    }
+
+    @Override
+    public void updatePassword(String email, String newPassword, int iterations, String newSalt) {
+
+    }
```

Comment:
Please remove it, because now it is not necessary. And from interface too. You should add this method when you will implement it.